tools/http3_test/src/runner.rs

```diff
@@ -140,6 +141,11 @@ pub fn run(
 
                 conn.on_timeout();
 
+                if conn.is_closed() && reqs_complete != reqs_count {
```

Comment:
FWIW, we already check for `conn.is_closed()` in two places below, so maybe can just add the logic there?

Reply:
makes sense!